rapids-metadata.json

```
@@ -227,6 +227,234 @@
           }
         }
       }
+    },
+    "24.10": {
+      "repositories": {
+        "_nvidia": {
+          "packages": {
+            "cubinlinker": {
+              "has_cuda_suffix": true,
+              "publishes_prereleases": false
+            }
+          }
+        },
+        "cucim": {
+          "packages": {
+            "cucim": {
+              "has_cuda_suffix": true,
+              "publishes_prereleases": true
+            }
+          }
+        },
+        "cudf": {
+          "packages": {
+            "cudf": {
+              "has_cuda_suffix": true,
+              "publishes_prereleases": true
+            },
```

```suggestion
            },
            "cudf-polars": {
              "has_cuda_suffix": true,
              "publishes_preleases": true
            },
```

Ok based on offline conversations with @vyasr and @raydouglass , I think we do want `cudf-polars` here (and in 24.08).